modes/company/evil-collection-company.el

```diff
@@ -39,6 +39,10 @@
 (defcustom evil-collection-company-supported-states '(insert replace emacs)
   "The `evil-state's which `company' function can be requested."
   :type '(repeat symbol))
+(defcustom evil-want-company-extended-keybindings nil
+  "The 'evil-company-extended' keybindings shoould be requested"
+  :type 'boolean
+  )
```

Comment:
move `)` 1 line up.

Reply:
You'll want the name to be something like "evil-collection-want-company-extended-keybindings".

Reply:
Thanks , adding changes in next commit